kuksa_databroker/databroker/src/grpc/server.rs

```diff
@@ -162,13 +163,22 @@ where
     Server::builder()
         .http2_keepalive_interval(Some(Duration::from_secs(10)))
         .http2_keepalive_timeout(Some(Duration::from_secs(20)))
-        .add_service(sdv::databroker::v1::broker_server::BrokerServer::new(
-            broker.clone(),
-        ))
-        .add_service(sdv::databroker::v1::collector_server::CollectorServer::new(
+        .add_service(
+            sdv::databroker::v1::broker_server::BrokerServer::with_interceptor(
+                broker.clone(),
+                authorization.clone(),
+            ),
+        )
+        .add_service(
+            sdv::databroker::v1::collector_server::CollectorServer::with_interceptor(
+                broker.clone(),
+                authorization.clone(),
+            ),
+        )
+        .add_service(kuksa::val::v1::val_server::ValServer::with_interceptor(
             broker.clone(),
+            authorization,
```

Comment:
@argerus - don't we need clone here as well?

Reply:
No, we don't use it (further) after this point.

(and if `clone()` was actually needed, it wouldn't compile)